.eslintrc.cjs

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  reportUnusedDisableDirectives: true,
+  parserOptions: {
+    sourceType: 'module',
+    ecmaVersion: 'latest'
+  },
+  overrides: [
+    {
+      // TODO: enable for `nextra-theme-blog` also
+      files: 'packages/nextra-theme-docs/**/*.{js,jsx,mjs,cjs,ts,tsx,mts,cts}',
```

Comment:
configured ESLint to forbid import from `next/link` and suggest using `<Anchor />` component that supports the both `experimental.newNextLinkBehavior` - `true` and `false`